docker/package/tezos_baking_wizard.py

```diff
@@ -283,6 +291,57 @@ def ledger_urls_info(ledger_urls):
     return info
 
 
+def search_baking_service_config(network, regex, default):
+    output = subprocess.run(
+        shlex.split("systemctl show tezos-baking-" + network + ".service"),
+        capture_output=True,
+    ).stdout
```

Comment:
Actually (related to my comment about getting two variables all over again), maybe there's no need to actually re-query `systemctl` at all? Get the stdout once, then just run regex queries on that. I haven't thought about this in depth, but seems like that should work :shrug:

Reply:
You're right

Reply:
Seems like you didn't do this :thinking: Did you decide it wasn't worth it?